python/singa/autograd.py

```diff
@@ -1115,7 +1161,8 @@ def __init__(self, kernel_size, stride=None, padding=0):
 
 
 class Tanh(Operation):
-
+    def __init__(self):
```

Comment:
if the init function does nothing, then no need to define it.

Reply:
to get names from base class operator, we need to use __init__()